lib/kernel/test/pg_SUITE.erl

```diff
@@ -317,228 +294,228 @@ thundering_herd(Config) when is_list(Config) ->
+    [peer:call(Peer1, ?MODULE, stop_proc, [Pid]) || Pid <- PidsToKill],
     sync(?FUNCTION_NAME),
-    sync({?FUNCTION_NAME, Peer1}),
+    sync({?FUNCTION_NAME, Node1}),
 
     Pids = lists:sort(Pids1 ++ Pids2 ++ Pids11),
     ?assert(lists:all(fun erlang:is_pid/1, Pids)),
 
-    disconnect_nodes([Peer1, Peer2]),
+    disconnect_nodes([Node1, Node2]),
 
     sync(?FUNCTION_NAME), %% Processed nodedowns...
     ?assertEqual([], lists:sort(pg:get_members(?FUNCTION_NAME, ?FUNCTION_NAME))),
 
-    [?assertEqual(ok, rpc(Socket2, pg, join, [?FUNCTION_NAME, ?FUNCTION_NAME, Pid])) || Pid <- Pids2],
-    [?assertEqual(ok, rpc(Socket1, pg, join, [?FUNCTION_NAME, second, Pid])) || Pid <- Pids11],
-    ?assertEqual(ok, rpc(Socket1, pg, join, [?FUNCTION_NAME, third, Pids11])),
+    [?assertEqual(ok, peer:call(Peer2, pg, join, [?FUNCTION_NAME, ?FUNCTION_NAME, Pid])) || Pid <- Pids2],
+    [?assertEqual(ok, peer:call(Peer1, pg, join, [?FUNCTION_NAME, second, Pid])) || Pid <- Pids11],
+    ?assertEqual(ok, peer:call(Peer1, pg, join, [?FUNCTION_NAME, third, Pids11])),
     %% rejoin
-    ?assertEqual(true, net_kernel:connect_node(Peer1)),
-    ?assertEqual(true, net_kernel:connect_node(Peer2)),
+    ?assertEqual(true, net_kernel:connect_node(Node1)),
+    ?assertEqual(true, net_kernel:connect_node(Node2)),
     %% need to sleep longer to ensure both nodes made the exchange
-    ensure_peers_info(?FUNCTION_NAME, [Peer1, Peer2]),
+    ensure_peers_info(?FUNCTION_NAME, [Node1, Node2]),
     ?assertEqual(Pids, lists:sort(pg:get_members(?FUNCTION_NAME, second) ++ pg:get_members(?FUNCTION_NAME, ?FUNCTION_NAME))),
     ?assertEqual(lists:sort(Pids11), lists:sort(pg:get_members(?FUNCTION_NAME, third))),
 
     {Left, Stay} = lists:split(3, Pids11),
-    ?assertEqual(ok, rpc(Socket1, pg, leave, [?FUNCTION_NAME, third, Left])),
-    sync({?FUNCTION_NAME, Peer1}),
+    ?assertEqual(ok, peer:call(Peer1, pg, leave, [?FUNCTION_NAME, third, Left])),
+    sync({?FUNCTION_NAME, Node1}),
     sync(?FUNCTION_NAME),
     ?assertEqual(lists:sort(Stay), lists:sort(pg:get_members(?FUNCTION_NAME, third))),
-    ?assertEqual(not_joined, rpc(Socket1, pg, leave, [?FUNCTION_NAME, left, Stay])),
-    ?assertEqual(ok, rpc(Socket1, pg, leave, [?FUNCTION_NAME, third, Stay])),
-    sync({?FUNCTION_NAME, Peer1}),
+    ?assertEqual(not_joined, peer:call(Peer1, pg, leave, [?FUNCTION_NAME, left, Stay])),
+    ?assertEqual(ok, peer:call(Peer1, pg, leave, [?FUNCTION_NAME, third, Stay])),
+    sync({?FUNCTION_NAME, Node1}),
     sync(?FUNCTION_NAME),
     ?assertEqual([], lists:sort(pg:get_members(?FUNCTION_NAME, third))),
-    sync({?FUNCTION_NAME, Peer1}),
+    sync({?FUNCTION_NAME, Node1}),
     sync(?FUNCTION_NAME),
 
-    stop_node(Peer1, Socket1),
-    stop_node(Peer2, Socket2),
+    peer:stop(Peer1),
+    peer:stop(Peer2),
     ok.
 
 nolocal(Config) when is_list(Config) ->
-    {Peer, Socket} = spawn_node(?FUNCTION_NAME, ?FUNCTION_NAME),
```

Comment:
I get why you want to rename the variable `Socket` to `Node` here, but could you in the future make that type of change into a separate commit? It makes it very hard to review the changes when 90% of all changes are renamed variables.

Reply:
My bad, I had a number of commits (one per test suite in fact), but then habit took over and I squashed the thing. For the future changes I will separate commits containing refactoring from logic changes.